From 7dc739d57d397427100639cebb1b8d1949e7a073 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Mon, 3 Jul 2017 18:25:01 -0700
Subject: [PATCH] MANTA-3339 muskie agent crash after invalid cueball state
 transition MANTA-3331 muskie logs are too verbose Reviewed by: David Pacheco
 <dap@joyent.com> Approved by: David Pacheco <dap@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 4b3a25d..021d247 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "2.2.3",
+        "cueball": "2.2.8",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
-- 
2.21.0

